tests/test_csv.nit

```diff
@@ -0,0 +1,34 @@
+import csv
+
+var words = "../examples/rosettacode/unixdict.txt".to_path.read_all.split('\n')
+
+var doc = new CsvDocument.empty
+for i in [0 .. 8[ do doc.header.add("Col{i}")
+
+var del = doc.delimiter.to_s
+var eol = doc.eol
+var sep = doc.separator.to_s
+for i in [0 .. 10000[ do
+	var ln = new Array[String]
+	for j in [0 .. 8[ do
+		var add_sep = if 100.rand >= 70 then true else false
```

Comment:
What about the following?

~~~nit
var add_sep = true
if true != if 100.rand >= 70 then true else false then add_sep = not add_sep
~~~